common/static/coffee/src/discussion/utils.coffee

```diff
@@ -113,7 +113,7 @@ class @DiscussionUtil
       alertDiv = $("<div class='modal' role='alertdialog' id='discussion-alert' aria-describedby='discussion-alert-message'/>").css("display", "none")
       alertDiv.html(
         "<div class='inner-wrapper discussion-alert-wrapper'>" +
-        "  <button class='close-modal dismiss' aria-hidden='true'><span class='icon fa fa-times' aria-hidden='true'></span></button>" +
+        "  <button class='close-modal dismiss' title='Close modal'><span class='icon fa fa-times' aria-hidden='true'></span></button>" +
```

Comment:
"Close modal" needs to be translated-- use gettext.

I also think I would just say "Close" or "Close dialog". I don't think "modal" is a very friendly user term.